Firebase/CoreDiagnostics/ProtoSupport/Protos/firebasecore.proto

```diff
@@ -65,28 +65,17 @@ message ICoreConfiguration {
   optional .logs.proto.mobilesdk.ios.ICoreConfiguration.PodName pod_name = 16;
   optional .logs.proto.mobilesdk.ios.ICoreConfiguration.ConfigurationType
       configuration_type = 1;
-  optional string version_name = 2;
```

Comment:
Do we want to use some sort of comment for each one mentioning those numbers could only be re-used for those fields?

Reply:
We could document that, but this is not the source-of-truth for this proto. Going forward, no changes should be made here that haven't already been made to the real proto. Adding fields here should be a matter of copy-pasting, not looking at the reserved values and choosing the next tag number.